weed/server/filer_server_handlers_write.go

```diff
@@ -197,6 +196,10 @@ func (fs *FilerServer) move(ctx context.Context, w http.ResponseWriter, r *http.
 // curl -X DELETE http://localhost:8888/path/to?recursive=true&ignoreRecursiveError=true
 // curl -X DELETE http://localhost:8888/path/to?recursive=true&skipChunkDeletion=true
 func (fs *FilerServer) DeleteHandler(w http.ResponseWriter, r *http.Request) {
+	if fs.option.WORM {
```

Comment:
We still need a way to delete a WORM file. Consider a file is half written, there is no way to delete it if WORM is enabled.

There are 2 possible options:
1. set up a retention, after the retention, the file is deleted automatically, or can be deleted by DELETE API here. 
2. Implement a new cmd, or force option (or privilieged option, which can be used for special user), wich careful control

Without above, the only thing we can do is to delete entry from filemeta table?

Reply:
add `?force=true`